src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedMonthGenitiveNames.cs

```diff
@@ -10,112 +10,30 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoAbbreviatedMonthGenitiveNames
     {
-        // PosTest1: Call AbbreviatedMonthGenitiveNames getter method should return correct value for InvariantInfo
         [Fact]
-        public void TestGetter()
+        public void AbbreviatedMonthNames_InvariantInfo()
         {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo,
-                    new string[] {
-                    "Jan",
-                    "Feb",
-                    "Mar",
-                    "Apr",
-                    "May",
-                    "Jun",
-                    "Jul",
-                    "Aug",
-                    "Sep",
-                    "Oct",
-                    "Nov",
-                    "Dec",
-                    ""
-                    },
-                    false);
+            Assert.Equal(new string[] { "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec", "" }, DateTimeFormatInfo.InvariantInfo.AbbreviatedMonthGenitiveNames);
         }
 
-        // PosTest2: Call AbbreviatedMonthGenitiveNames setter method should return correct value
         [Fact]
-        public void TestSetter()
+        public void AbbreviatedMonthNames_Set()
         {
-            VerificationHelper(new DateTimeFormatInfo(),
-                    new string[] {
-                    "1",
-                    "2",
-                    "3",
-                    "4",
-                    "5",
-                    "6",
-                    "7",
-                    "8",
-                    "9",
-                    "10",
-                    "11",
-                    "12",
-                    ""
-                    },
-                    true);
+            string[] newAbbreviatedMonthGenitiveNames = new string[] { "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "" };
```

Comment:
> I was told off for adding tests in a precious refactoring PR

If you want to add additional test cases, that's fine, but please do it in a separate commit.  When reviewing a refactoring, we need to be able to correlate the old with the new, and that because much more difficult when meaningful test changes are mixed in.